Week-10/url-shortner/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "url-shortner",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/material": "^5.10.4",
+    "@mui/x-data-grid": "^5.17.1",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
do we really need testing dependency ?, remove additional dependency from package.json